### DISCUSSION TITLE
GDI image no longer supported for NAOMI

### DISCUSSION THREAD

**Discussion by pizzzza19** [flyinghead/flycast#1779] [Q&A] [answered]
It looks NAOMI / NAOMI2 GD-ROM titles won't boot up by GDI format image since 2024 summer binary.
Currently, Dreamcast ROMs both boot with GDI and CHD but NAOMI doesn't.
In many cases CHD image gets smaller size so most popular way to go. I just want to know why Flycast no longer support NAOMI GDI image?

**Comment by flyinghead** [comment, marked as answer]
I didn't think anybody was using GDI files for naomi titles. I couldn't find any reference of a mameset using GDI files but perhaps I didn't look hard enough.
This was done to simplify the code when I added support for Android scoped storage.

**Reply by pizzzza19** [reply]
I see. It’s a major file format by MAME. CHD is a single file and lossless. Good to hear that simplify to support only CHD formats.
I don’t see any of detail Flycast supported ROM format guides but now everything clear. Thanks.
